docs/installation/install-binary.md

```diff
@@ -110,228 +106,703 @@ scp /usr/local/sbin/* karmada-02:/usr/local/sbin/
 scp /usr/local/sbin/* karmada-03:/usr/local/sbin/
 ```
 
+# Generate Certificates
 
+## Step 1: Create Bash Scripts and Configuration Files
 
-## Generate certificate
+The following scripts will generate certificates using the `openssl` command. 
 
-Generated using the `openssl` command. Note yes `DNS` and `IP` when generating the certificate.
+We separate CA & leaf certificates generation script, so when you need to change Subject Alternative Name of leaf certificates (aka Load Balancer IP), you can reuse CA certificates, and run generate_leaf.sh to generate leaf certificates only.
 
-Execute operations at `karmada-01`.
 
-#### create a temporary directory for certificates
 
-```bash
-mkdir certs
-cd certs
+There are 3 CAs: front-proxy-ca, server-ca, etcd/ca. Why we need 3 CAs please see: [PKI certificates and requirements](https://kubernetes.io/docs/setup/best-practices/certificates/), [CA Reusage and Conflicts](https://kubernetes.io/docs/tasks/extend-kubernetes/configure-aggregation-layer/#ca-reusage-and-conflicts)
+
+If you use etcd provided by users, you can ignore `generate_etcd.sh` and `csr_config/etcd`.
+
+
+
+The file structure should look like this:
+
+```
+├── csr_config													# certificate signing request config file.
```

Comment:
should we create this directory in karmada repo and put these scripts inside?

Reply:
This scripts is more like a building block for Physical Machine Deployment. I suspect most audience will need to modify these to their Deployment System's requirements. 

These scripts are really not related to Karmada main repository. I am afraid putting these scripts there will burden the maintainer of karmada-io/karmada.

I think it will be better if I can create a directory in karmada-io/website repository, and put all these scripts there, or upload as an attachment.

Reply:
By the way, putting all the scripts inside a git repository will make the users much easier to clone them.

Reply:
cc @Poor12

Reply:
How about putting it in/docs/resources? You can create a directory about installation/install-binary.

Reply:
> How about putting it in/docs/resources? You can create a directory about installation/install-binary.

OK. This looks like an appropriate location for attachments.